etc/ci/before_deploy.sh

```diff
@@ -162,7 +162,7 @@ Description: Syntax highlighter for git
  detection, and restructured diff output for git on the command line.
 EOF
 
-    fakeroot dpkg-deb --build "$tempdir" "${dpkgname}_${version}_${architecture}.deb"
+    fakeroot dpkg-deb -Z xz --build "$tempdir" "${dpkgname}_${version}_${architecture}.deb"
```

Comment:
Please use `-Zxz` without a space.

This is what man page says. (`-Z xz` works - I just tested it, but is not explicitly documented).

Reply:
OK, changed! Personally I have always inserted a space after a single-letter flag for every Unix/Linux command I've used in the last 20 years as far as I remember -- I think a Linux command would be pretty broken if it didn't accept a space. Plus one has to remember that Java and Windows people for some reason think it's acceptable to use a single `-` with the long option variants :)